lint.swift

```diff
@@ -0,0 +1,547 @@
+            "init(\n    x: Int,\n    y: Int\n) { }",
+            "func foo2bar()  { }",
+            "func foo2bar(x: Int, y: Int)  { }",
+            "func foo2bar(\n    x: Int,\n    y: Int\n) {\n    \n}"
+        ],
+        nonMatchingExamples: ["init() { /* comment */ }", "init() {}", "func foo2bar() {}", "func foo2bar(x: Int, y: Int) {}"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: "$declaration {}",
+        autoCorrectExamples: [
+            ["before": "init()  { }", "after": "init() {}"],
+            ["before": "init(x: Int, y: Int)  { }", "after": "init(x: Int, y: Int) {}"],
+            ["before": "init()\n{\n    \n}", "after": "init() {}"],
+            ["before": "init(\n    x: Int,\n    y: Int\n) {\n    \n}", "after": "init(\n    x: Int,\n    y: Int\n) {}"],
+            ["before": "func foo2bar()  { }", "after": "func foo2bar() {}"],
+            ["before": "func foo2bar(x: Int, y: Int)  { }", "after": "func foo2bar(x: Int, y: Int) {}"],
+            ["before": "func foo2bar()\n{\n    \n}", "after": "func foo2bar() {}"],
+            ["before": "func foo2bar(\n    x: Int,\n    y: Int\n) {\n    \n}", "after": "func foo2bar(\n    x: Int,\n    y: Int\n) {}"]
+        ]
+    )
+
+    // MARK: EmptyTodo
+    try Lint.checkFileContents(
+        checkInfo: "EmptyTodo: `// TODO:` comments should not be empty.",
+        regex: #"// TODO: ?(\[[\d\-_a-z]+\])? *\n"#,
+        matchingExamples: ["// TODO:\n", "// TODO: [2020-03-19]\n", "// TODO: [cg_2020-03-19]  \n"],
+        nonMatchingExamples: ["// TODO: refactor", "// TODO: not yet implemented", "// TODO: [cg_2020-03-19] not yet implemented"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles]
+    )
+
+    // MARK: IfAsGuard
+    try Lint.checkFileContents(
+        checkInfo: "IfAsGuard: Don't use an if statement to just return – use guard for such cases instead.",
+        regex: #" +if [^\{\n]+\{\s*return\s*[^\}]*\}(?! *else)"#,
+        matchingExamples: [" if x == 5 { return }", " if x == 5 {\n    return nil\n}", " if x == 5 { return 500 }", " if x == 5 { return do(x: 500, y: 200) }"],
+        nonMatchingExamples: [" if x == 5 {\n    let y = 200\n    return y\n}", " if x == 5 { someMethod(x: 500, y: 200) }", " if x == 500 { return } else {"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles]
+    )
+
+    // MARK: LateForceUnwrapping3
+    try Lint.checkFileContents(
+        checkInfo: "LateForceUnwrapping3: Don't use ? first to force unwrap later – directly unwrap within the parantheses.",
+        regex: [
+            "openingBrace": #"\("#,
+            "callPart1": #"[^\s\?\.]+"#,
+            "separator1": #"\?\."#,
+            "callPart2": #"[^\s\?\.]+"#,
+            "separator2": #"\?\."#,
+            "callPart3": #"[^\s\?\.]+"#,
+            "separator3": #"\?\."#,
+            "callPart4": #"[^\s\?\.]+"#,
+            "closingBraceUnwrap": #"\)!"#
```

Comment:
Codacy found an issue: [Multi-line collection literals should have trailing commas.](https://app.codacy.com/gh/Flinesoft/HandySwift/pullRequest?prid=5730214)